### ISSUE TITLE
Chicago Research Report 

### ISSUE THREAD

**Issue by LauraReynal** [MozillaFoundation/plan#372]
Release field report from Chicago research trip. :earth_americas: :

RACI

Phase: Research / Build / Ship
Owner: @LauraReynal
Decision: @davidascher 
Lead design:@cassiemc
Lead dev: n/a

**Comment by LauraReynal** [comment]
Target is May 8th for first version of the report. 
Adding @sabrinang in the issue to follow.

**Comment by OpenMatt** [comment]
needs design resource for week of May 4

**Comment by OpenMatt** [comment]
@cassiemc will review

**Comment by cassiemc** [comment]
Hey @LauraReynal @thisandagain – Earlier this week I thought @sabrinang might have time to help but she is on PTO 4/5 days next week. So please lean on me to help with this, especially after Monday (busy day onboarding Jordan and our new contractor). When do you think the content for this will be ready to go?

**Comment by LauraReynal** [comment]
Hey @cassiemc thanks for the message. I am unsure when the content will be ready for a design review, but it will be probably not be before the end of this heartbeat May8th. 
Will let you know a couple days in advance when ready for design. 

However, there is a bunch of design help needed on some graphics. They would need to happen before the report itself in order to incorporate them. What do you think ? 
tracking here the design tasks have Sabrina's handle https://github.com/MozillaFoundation/research/issues/47

**Comment by cassiemc** [comment]
hey @LauraReynal, @thisandagain filled me in on what needed to be done last week and I think this is a good first task for Jordan. Not sure his Github nick yet but @thisandagain is working with him 1:1 in Portland.

**Comment by thisandagain** [comment]
Great. Thanks @cassiemc 

Added @lovegushwa as the assignee for the design tickets that were assigned to Sabrina. @LauraReynal perhaps you and @lovegushwa could connect tomorrow morning to talk through the illustrations and diagrams? It would also be nice to have him join us for the draft walkthrough on Thursday.

**Comment by OpenMatt** [comment]
@LauraReynal status? [[flagged!]](http://plzkthx.herokuapp.com)

**Comment by LauraReynal** [comment]
Work in progress, moving to next heartbeat.

**Comment by OpenMatt** [comment]
* This is being deprecated -- @LauraReynal will update w. more on timelines

**Comment by LauraReynal** [comment]
This will be tackled after Bangladesh and Whistler.

**Comment by OpenMatt** [comment]
@LauraReynal: status? I'm going to move this to July 17 -- correct if that's wrong.

**Comment by cassiemc** [comment]
@LauraReynal is on PTO this week and next so let's bump to the following heartbeat.

**Comment by thisandagain** [comment]
Update: This took a backseat to the Gates Foundation proposal. Picking up in the next heartbeat.

**Comment by LauraReynal** [comment]
Update: made good progress on copy - still needs some work. 

For next heartbeat, could we get help from: 

- @lovegushwa on design for graphics and the report itself 

- @thisandagain content review - mid heartbeat 

- also would like to get a review from @omnignorant if possible 

- @adamlofting for pulling out a couple of graphics from our particpants data base (when available: files are ready)

- @KevZawacki for a copy review (late next heartbeat) on the report + on the blog post (final)

Thoughts ?

**Comment by KevZawacki** [comment]
I can definitely do copy review/writing.

**Comment by lovegushwa** [comment]
@LauraReynal 
I want to help with design / fixing up the template a bit more as well.

**Comment by cassiemc** [comment]
WFM if @lovegushwa can balance this with the Fundraising work. There is a fundraising workweek this week so his involvement will be minimum at this point. Have updated the RACI above.

**Comment by LauraReynal** [comment]
Update : 
- 90% of the copy is done - Im still editing but the most important sections are written. Needs work for the rest such as user portraits, conclusion and appendixes. 
- @thisandagain will do a full content review. 
- Sending to @KevZawacki for copy and language review on monday 10th 

Next week: There will be a lot to do and the week will be dedicated to editing frameworks, layout and copy revisions. I'll be working with Jordan but I already started. 

We should be good to publish on Wednesday 19th or Thrusday 20th.

**Comment by LauraReynal** [comment]
<img width="517" alt="screen shot 2015-08-07 at 18 57 41" src="https://cloud.githubusercontent.com/assets/8261945/9142227/37c23f1c-3d36-11e5-9a93-cbe0169f280a.png">

**Comment by LauraReynal** [comment]
cc/ @davidascher

**Comment by LauraReynal** [comment]
FYI, design work is tracked here https://github.com/MozillaFoundation/Design/issues/71

**Comment by LauraReynal** [comment]
:mag:  @adamlofting Do you think you would have time to convert the spreadsheet into a couple of pie charts ? Or should I do it manually as for the previous reports? Let me know, im happy to remove you from this bug if you have no time.

**Comment by LauraReynal** [comment]
@KevZawacki the document is ready for your copy review from page 1 to 48 now if you would like to start. I can ping you later when the rest of the copy is finished. Let me know if this is okay for you :santa: :-)

**Comment by LauraReynal** [comment]
Update for the universe: 
* Just finished writing the recommendations, now ready for review. Page 56 to 61 :sparkles: 
* @anmechung I pinged you inside the document for a check on the library bits, as per our discussion.
* Jordan and I are working on the design at the same time -- because these things take time.
<img width="627" alt="screen shot 2015-08-12 at 14 48 35" src="https://cloud.githubusercontent.com/assets/8261945/9225418/3a957460-4101-11e5-96fe-504f3851cee7.png">

Left to do:
* Write the personas 
* Write the appendixes 
* Further work on the frameworks

**Comment by LauraReynal** [comment]
Update: 
The document has been fully reviewed by Andrew, it's good to go. We've made some good design work with Jordan. I'm meeting with Adam monday. I still have a bunch to do, but I think we can reach our deadline of Wednesday or thrusday the latest. 
I have started to work on the inDesign document. 
:racehorse: 

@KevZawacki I saw you made some copy edits. Thank you very much. Is the rest of the document good to go, or is there a page I should stop at, when I migrate to inDesign?

**Comment by KevZawacki** [comment]
Hey @LauraReynal, I'm copyediting pages 20 and onward right now. Will be done before 5 p.m. ET!

**Comment by LauraReynal** [comment]
We have made some cool progress with @adamlofting today. Check out some graphs below :-) the rest will be in the report. 
![level of engagement with the web and computer ownership](https://cloud.githubusercontent.com/assets/8261945/9309960/bfedb6ac-4504-11e5-9ba5-8b3c5ff3955f.png)


![web journey](https://cloud.githubusercontent.com/assets/8261945/9309958/bfed4d84-4504-11e5-9982-bc4b69c9591b.png)
![os distribution](https://cloud.githubusercontent.com/assets/8261945/9309971/d9f8a5fc-4504-11e5-9cbe-862238787893.png)
![who we talked to](https://cloud.githubusercontent.com/assets/8261945/9309959/bfed4b72-4504-11e5-9b48-dfd1113f49eb.png)

**Comment by LauraReynal** [comment]
Update: I have started to work on the layout in inDesign, but the document needs a full copy review before possible. 
@KevZawacki has been pulled out because of emergencies with Webmaker Launch so this project is likely to be delayed. 

Will update regularly on the progress. Thanks !

**Comment by LauraReynal** [comment]
cc/ @secretrobotron here is the ticket to update if you have time to help on copy edits

**Comment by cassiemc** [comment]
@LauraReynal I could do some proofreading this afternoon if you can link me to the latest doc? Realize I might've missed my window with you today!

**Comment by LauraReynal** [comment]
Update: 

- @KevZawacki has finished copy reviewing the whole document. HUGE thanks :bow: 

- I just finished designing the whole document

Next Step: 

- @lovegushwa will do a final design review. 

- Write a blog post and publish, 

<img width="509" alt="screen shot 2015-08-25 at 22 13 52" src="https://cloud.githubusercontent.com/assets/8261945/9479809/fface268-4b76-11e5-9a3c-a8823fdd27a0.png">

**Comment by LauraReynal** [comment]
@KevZawacki would you mind proofing / editing this blog post for publishing the report please? 
https://etherpad.mozilla.org/blogpostchicago

**Comment by LauraReynal** [comment]
@davidascher @cassiemc @lovegushwa The final document is here, in PDF.
 https://drive.google.com/open?id=0B0X05aIHuAbec1htMHplMHpuYk0 

I am ready to publish as soon as I have your GO, if you'd like to check it out before we send.

**Comment by KevZawacki** [comment]
@LauraReynal No problem -- just proofed. Good to go!

**Comment by LauraReynal** [comment]
Thanks @KevZawacki 

Here is the latest link for the report https://drive.google.com/open?id=0B0X05aIHuAbeR0RZTUx0cERWVGs

**Comment by LauraReynal** [comment]
@cassiemc here is the latest link

**Comment by cassiemc** [comment]
Great, I'm reviewing now.

**Comment by davidascher** [comment]
I'm reading it too.  First question I have is whether we really want to
call it Chicago Report.  I understand why we've been using that moniker
internally, but I don't think it's a useful title to someone with no
context.

I'll come up with some more specific feedback shortly.

Do we have any institutional audiences or partners who expect this in a
particular time frame?

**Comment by cassiemc** [comment]
I had the same thought about the title @davidascher. Also, I think we should deliver digital pdfs as page-by-page rather than as spreads, but realize that might mean rejigging some of the design.

**Comment by LauraReynal** [comment]
I agree with the title.  It does not make much sense.  Let me know if you have many ideas. 
We don't have any institutional partner waiting on this. 

@cassiemc it has been designed with the idea that it would be read with spreads. For example,  full page pictures and quotes are facing relevant text etc.  Maybe we can switch to single pages for the next one? 
Or is it really a blocker for this one? Let me know.

**Comment by xmatthewx** [comment]
If your spreads in InDesign are one wide page, it will likely be a huge pain to reformat everything. I don't think it's worth it.

If your spreads in InDesign are two pages, you can export it as single pages and then configure the PDF to display spreads by default. Users can override it to view single pages, which can make reading easier, particularly on mobile. (True for Adobe and some mobile readers. Not sure about in-Browser PDFs)

**Comment by cassiemc** [comment]
I assumed they were 2 pages and could be exported as single but that would take some effort yet to split the designs properly (I could help). If other reports were spreads I'd say don't worry about it and we can fix it next time.

**Comment by LauraReynal** [comment]
They are 2 pages and they are separable,  but as you point out it would be a lot of work.  Let's think about it for the next ones if okay with you.

**Comment by LauraReynal** [comment]
@cassiemc Feel free to use the inDesign file directly, if you want to try stuff or make direct changes https://drive.google.com/open?id=0B0X05aIHuAbeRlpCM0ZoejVXN0k

**Comment by cassiemc** [comment]
ok thanks @LauraReynal !

**Comment by davidascher** [comment]
Overall, lots of great stuff in here!

Biggest critique I have is that needs a thorough copyediting pass by a
native english speaker, paying careful attention to gallicisms.  Things
like the use of "evolve" and "mitigate" which are used differently in
French and English.

It's also quite long, although we can probably just deal with that in a
separate, summary document.

I have some typographic nits, but the biggest one is that the light
typeface is basically unreadable when printed on a home or low-end office
printer.

Some of the quotes are amazing and powerful. I'd like to see them displayed
on the Mozilla office screens.

Some random notes:

p. 6 - curly quotes instead of straight quotes
p. 6 - remove leading space before "Our approach"
p. 10 change "evolve" in second paragraph -- I believe that's a gallicism.
Same comment about page 25, 50.
p. 13 I don't understand the "flavors of blur" heading.
p. 14 titling font seems off-template
p. 14 "different between users in emerging/mobile-first markets and
developed markets"
p. 18 "We then combined the resulting list with our data" -- what data?
p. 18  whether skills "operate as bridges" feels like a claim that is worth
unpacking somehow.
p. 22 remove "In Chicago" in second paragraph.
p. 22 -- the typeface in the small subtitles in the pull quotes print too
lightly on my fairly high end printer.  We should make sure the report is
printable by normal printers.
p. 34 chicago should be Capitalized
p. 40 -- Don't think "mitigated" is the right word.  That paragraph is a
bit hard to understand.
p. 40 -- Not Kick, Kik.
p. 40 -- first paragraph in second column is repetitive with second para in
first column
p. 41 first line needs a rewrite


[david's getting bored with keeping detailed notes at this point]

**Comment by davidascher** [comment]
By the way -- once we've done the style copyedit, I'm going to make it
mandatory reading for my entire team and recommended reading for all of
MoFo.  There's a large number of lessons worth reading, digesting and
discussing.

**Comment by chrislarry33** [comment]
@davidascher I will do same with my team, maybe unpack it in a Socratic vidyo seminar? Itching to read myself but waiting on final copy.

**Comment by davidascher** [comment]
Yes finding a way to get people engaged in the inquiry would be great.

**Comment by KevZawacki** [comment]
@davidascher & @LauraReynal I'm happy to do a second, more thorough edit to smooth out the gallicisms (first copyedit I was looking for big items).

At ~10 pgs a day, OK to have the next review completed by 9/4?

@LauraReynal, what's the best way to edit the copy now that it's laid out?

**Comment by davidascher** [comment]
My own take is quality trumps schedule. Sep 4 seems fine to me but I don't
know where we're waiting for this work.

**Comment by LauraReynal** [comment]
@davidascher Thanks for the review and the detailed notes. 

* Title: agree that we need to find a new title for the report, will think of some ideas and post here 

* Design: Ok for increasing the thin typography and the other changes you suggest. 

* Yes to a socratic seminar @chrislarry33 - we can schedule it once it's finished. 

* @KevZawacki OK with the copy edit by Sept 4th -- Would you be able to do it directly in the inDesign document ? I think you could take the trial version, if you dont have the software yet. Let me know if this is comfortable for you. Below is the list that I have already corrected from David's. I left the copy ones for you. 

> p. 6 - curly quotes instead of straight quotes 

I dont know how to change this @lovegushwa @cassiemc 
> p. 6 - remove leading space before "Our approach" 

:white_check_mark: done 
> p. 10 change "evolve" in second paragraph -- I believe that's a gallicism. Same comment about page 25, 50.  

for @KevZawacki 
> p. 13 I don't understand the "flavors of blur" heading. 

@davidascher this is from Bill Rankin from Radical Cartography. I am afraid I cannot change the title of his work.
> p. 14 titling font seems off-template 

 @lovegushwa @cassiemc What do you suggest ?
> p. 14 "different between users in emerging/mobile-first markets and developed markets" 

@KevZawacki 
> p. 18 "We then combined the resulting list with our data" -- what data?  

:white_check_mark: changed to "our observations".

> p. 18 whether skills "operate as bridges" feels like a claim that is worth unpacking somehow. 

ok 
> p. 22 remove "In Chicago" in second paragraph. 

:white_check_mark: done 
> p. 22 -- the typeface in the small subtitles in the pull quotes print too lightly on my fairly high end printer. We should make sure the report is printable by normal printers. 

@lovegushwa should we use book instead of thin ? 
> p. 34 chicago should be Capitalized 

:white_check_mark: done 
> p. 40 -- Don't think "mitigated" is the right word. That paragraph is a bit hard to understand. 

@KevZawacki 
> p. 40 -- Not Kick, Kik. 

:white_check_mark: done
> p. 40 -- first paragraph in second column is repetitive with second para in first column 

:white_check_mark: Corrected 
> p. 41 first line needs a rewrite

@KevZawacki

**Comment by LauraReynal** [comment]
Also, if @KevZawacki and @cassiemc and / or @lovegushwa are  all working on the inDesign file, we should do a quick check so that we don't overwrite each others and work on different versions of the same doc. 

@cassiemc let us know if you are making design changes directly, so that we can see how best to organize each others.

**Comment by cassiemc** [comment]
@LauraReynal @KevZawacki I won't touch the inDesign file until the copy edits are done if that's easier. I haven't made any changes yet. @KevZawacki are you able to edit the indesign file directly? if so, maybe you can just let me know when you're done with a set of pages then I'll get started with design tweaks.

(fyi @lovegushwa will need to concentrate on the EOY and other engagement work so I am going to happily take over his work on this) :)

**Comment by KevZawacki** [comment]
@cassiemc -- Agreed, let's do copy first. I'll be editing ~15 pages a day. Should I ship the updated file at the end of each day?  

@LauraReynal -- I have InDesign and can edit directly. Can you share the latest .indd file? Thanks!

**Comment by LauraReynal** [comment]
@KevZawacki in this folder, you will have everything 
https://drive.google.com/a/mozilla.com/folderview?id=0B0X05aIHuAbed2xfX0Z5c09KaTg&usp=sharing

Research Reports Assets > Chicago Report April 2015 > Chicago_X Folder > Look for the inDesign file called 27-08-15

Please save a new version of the document each day when you close it, with the date. You can see the different versionning we have (I had dramatical moments with Gdrive before, so we are careful now).

**Comment by KevZawacki** [comment]
@LauraReynal @cassiemc I've finished edits on pages 1 - 39, and uploaded to our [shared drive](https://drive.google.com/drive/folders/0B2qu3ExKWIlvUEdMdHRlNFU1Zzg); the doc is titled "CHICAGO_X -31-08-15.indd."

Let me know if I should make any changes to this process. Otherwise, I'll upload a new doc EOD tomorrow with more edits.

**Comment by LauraReynal** [comment]
@KevZawacki this process is perfect :-) thanks !

**Comment by KevZawacki** [comment]
Hi @LauraReynal and @cassiemc, I've finished edits on pages 1 - 97 and uploaded to our shared drive (doc is titled "CHICAGO_X -02-09-15.indd").

**Comment by cassiemc** [comment]
@LauraReynal @KevZawacki  I had to request Creative Cloud. I'm due for an update anyway, but if either of you get a chance to save it down to CS6 that would be helpful (I will have more time to work on this tomorrow, today is chocker-block with 1:1s)

**Comment by KevZawacki** [comment]
@LauraReynal, pages 1 - 127 are now edited and uploaded to the [shared drive](https://drive.google.com/drive/folders/0B2qu3ExKWIlvUEdMdHRlNFU1Zzg). The latest file is titled "CHICAGO_X-03-09-15.indd."

A few items that require your attention:

+ On page 74, you write, "We expand on the perception of learning to code on page XX." Can I change "XX" to "104"? (Page 104 is the section "Learning code is not appealing.")

+ The same two paragraphs are repeated on pages 82 and 90 ("As we studied the Web Journey in both Chicago and emerging countries, similarities and differences came to light...") Is this intentional? 

+ On page 128 ("Deep dive on informal educators"), you use the same language you do on page 118 ("Deep dive on teenagers"). Should I simply change all instances of "teenager" on page 128 to "informal educators"?

@cassiemc, I've also uploaded "CHICAGO_X-03-09-15.idml." Does that do the trick?

**Comment by cassiemc** [comment]
Yes that worked, thanks @KevZawacki!

**Comment by KevZawacki** [comment]
Hi @cassiemc, I'm planning to finish copy edits today. Is there a updated file from your end I should use?

**Comment by cassiemc** [comment]
No, I opened the file but didn't do anything with it, so go ahead! I am in
Toronto today so probably won't have much time in the end, will start in
earnest next week if you are planning to finish today. Then I can make any
additional copy changes if needed. Thanks Kevin!

**Comment by LauraReynal** [comment]
@KevZawacki 

* page 74 - yes, thank you for catching that ! 
* page 82 and 90 - no it's not intentional, feel free to edit how you want.
* p.128, yes that is a good idea - thanks :+1: 

Let me know if there  is anything else I can help with

**Comment by KevZawacki** [comment]
Hi @LauraReynal and @cassiemc, I've uploaded the fully-edited report to our shared drive. There are two versions:

+ CHICAGO_X -04-09-15.indd
+ CHICAGO_X -04-09-15.idml

Let me know if/how I can help further!

**Comment by cassiemc** [comment]
:hand::hand:Woo, nice work @KevZawacki! I'll dig into finalizing the design when we're back on Tuesday. Thanks for saving out two versions, I haven't gotten my creative cloud yet.

**Comment by LauraReynal** [comment]
Huge thanks for the work @KevZawacki 
Aside from @cassiemc 's design pass, the last thing we need to do is find a new title for the reportS.
I think we should use a title that can work for all the reports in the future, with spin-off.

Some first ideas: 
- Mozilla Foundation Research

**Comment by KevZawacki** [comment]
@LauraReynal I like the first one. Or, something similar like: 

Mozilla Foundation Research
Global Web Literacy Studies: Chicago Report

**Comment by LauraReynal** [comment]
@KevZawacki it's perfect ! 

@cassiemc you want to make the edit on Chicago's document ? 

I will update the other reports as well for coherence.

**Comment by LauraReynal** [comment]
@cassiemc I will wait to see your design pass on chicago's doc, because the layout is different for the other reports. For now I have something like this 
<img width="557" alt="screen shot 2015-09-08 at 13 43 27" src="https://cloud.githubusercontent.com/assets/8261945/9735026/a24bb0d6-562f-11e5-964a-034f7cdffd94.png">

**Comment by cassiemc** [comment]
Hey @LauraReynal @KevZawacki I can definitely make the edit in the doc. Today is crazy with meetings again but I should finally have a good stretch of time to open and tinker with this tomorrow. I did notice a few links are missing upon open. Can either of you do a "package" export of the file and upload that whole folder to drive? (make sure it includes all linked files)

Re. the title, what do you guys think of removing the word 'report'. So:

Mozilla Foundation Research
Global Web Literacy Studies: Chicago ~~Report~~

Global Web Literacy Research: Chicago

**Comment by cassiemc** [comment]
(I think I am wrestling with the Mozilla logo along with the Mozilla Foundation in the title – somewhat redundant and  from an outsider pov, seems unnecessarily reflective of our internal organization.)

**Comment by jgmac1106** [comment]
Just food for thought Would Universal Web Literacy Research: Chicago as a
title align better with the current mission? If the goal is to spark a more
geographic meaning Global Web Literacy Research sounds better.

**Comment by LauraReynal** [comment]
@cassiemc i love your option 2 : 

Global Web Literacy Research: Chicago
Mozilla Foundation

@jgmac1106 I would lean towards Global for the geographic connotation.

**Comment by LauraReynal** [comment]
https://drive.google.com/open?id=0B0X05aIHuAbeV3RCVzJQWWNJdE0
@cassiemc here is the package

**Comment by cassiemc** [comment]
@LauraReynal Do you have more photos from Chicago than are listed here? https://www.flickr.com/photos/laura_nk/albums/72157656542031096

**Comment by LauraReynal** [comment]
@cassiemc I shared the whole Chicago folder with you it's also here https://drive.google.com/a/mozilla.com/folderview?id=0B0X05aIHuAbeTjhqbS1NVWNSazg&usp=sharing
Inside you will find a folder called "photos' which have more things than Flickr

**Comment by LauraReynal** [comment]
I was not aware of the document page size, feel free to change it

**Comment by LauraReynal** [comment]
@cassiemc I just shared with you the whole research reports folder. 
I added for you the .svg and .ai of the iterative design: 
Research Reports Assets > _Graphics

**Comment by cassiemc** [comment]
beautiful thanks @LauraReynal!

**Comment by cassiemc** [comment]
This is taking me a lot longer than expected! Still, I expect to finish today / tonight.

**Comment by cassiemc** [comment]
As I was going through the file tonight I started to realize there were lots of pages that were wonky, probably due to my older version of indesign. I finally got my service now request for creative cloud this week and am installing now. I have made lots of updates to the photographs, colors, paragraph styles and the grid on master pages – including updating the document size so it prints better – really important first steps.

But rather than spend my whole weekend on this, I am going to update the milestone. So sorry @LauraReynal and @kevinzawacki -- I am blocking off all but an hour next Tuesday to finish this.

**Comment by cassiemc** [comment]
Here are pages 1-40, should be a decent structure now so editing the other sections goes a bit quicker. https://drive.google.com/a/mozilla.com/file/d/0BzfN9sXlOXt1VjRzS1ozQ1RjY3M/view?usp=sharing
@LauraReynal, eager to hear what you think.

**Comment by davidascher** [comment]
Looks great.

I just got the package from Laura w/ the first three reports, and one
comment about them is that they seem oversize given the amount of copy on
each page.  Type size seems really large and information density feels off
for so much printed weight.

Cassie -- thinking about this chicago report, any thoughts about whether a
less than 8.5x11 layout could work (without needing to relayout)?  I find
myself unable to gauge some of this stuff w/o actually printing them out.

--da

**Comment by LauraReynal** [comment]
@cassiemc thanks, looks great.
What do you think is the time estimate to publish it ? Let me know if I can help.

@davidascher @cassiemc yes, we messed up when printing the previous reports. They were designed to be smaller, but they got printed in A4. 
If we do decide to print Chicago's edition, then we will do some tests and print in smaller dimension.

**Comment by LauraReynal** [comment]
@cassiemc on page 8, the title "Chicago urban landscape" is a different style than the other titles. 
It's the same level that "the Web journey as a framework" and others.

**Comment by cassiemc** [comment]
Hey @LauraReynal let's look at the whole thing once it's all together, I was trying a few different headlines to see if the design might help highlight the rhythm of the content. I'm starting to understand the content structure much better now, so I'm excited about how different design treatments can help tell the story (need to balance that with consistency though, you're right). I'm planning for the whole doc to be finished end of this week so we can demo it. Does that work for you? 

@davidascher Re. the size, two thoughts – I love the idea of reducing paper waste, +1. So am looking forward to getting these living online. Second thought – @LauraReynal Do we mostly print these ourselves and mail out to different offices (not sure where they are currently distributed but we should put them everywhere!), or do people print their own? Reducing the size or putting more than one page on a sheet means more work for someone if they print their own, rather than print out at letter or A4 and staple it together. 

(Note: I'm designing for Letter size now, with the understanding that we are favoring the North American audience for this report; people can still print A4 but it'll have a white bar on the top or bottom)

It's a significant amount of work to change the document size at this point. It means changing all the text sizes and probably rejigging layouts, so I'd say we bear this in mind for the next report.

**Comment by LauraReynal** [comment]
@cassiemc ok for the titles, no problem. Feel free to do as you want. It's great !

@cassiemc Ok for the timeline. Would thursday EOD work for you ? This way we could ship it on friday with a blog post. 

Printing:
We have printed them, and shipped many are in different offices. (Toronto, SF, NY, Portland, Vancouver, Berlin...) Also in the hands of some partners. Usually people dont print their own.

**Comment by cassiemc** [comment]
@LauraReynal What do you think about breaking out the deep dive on teenagers and educators into a separate pub? I made it through another 50 pages today (will upload later), but tomorrow is busier for me so I don't see getting it done by Thursday, otherwise.

**Comment by cassiemc** [comment]
Pages 1-100: https://drive.google.com/a/mozilla.com/file/d/0BzfN9sXlOXt1OTRpU2xTenNjbFk/view?usp=sharing

**Comment by cassiemc** [comment]
(Note: none of the page headlines are correct; I have a running list of things to go back and do at the end, including final typesetting, checking consistency, etc)

**Comment by LauraReynal** [comment]
> @LauraReynal What do you think about breaking out the deep dive on teenagers and educators into a separate pub? I made it through another 50 pages today (will upload later), but tomorrow is busier for me so I don't see getting it done by Thursday, otherwise.

Im not unsure I understand "pub" = publication ? you mean breaking out the graphics or putting them into other reports? 

I would leave them as such for this one. When we bring the reports online and build the research website, I was thinking about a section for personas/segmentation'/journeys etc.

**Comment by LauraReynal** [comment]
wow the design is really beautiful @cassiemc :blush: 


One thing im wondering is this page; the warm yellow from the title and the acid yellow from snapchat are hard to look at together. 
<img width="1295" alt="screen shot 2015-09-17 at 08 14 53" src="https://cloud.githubusercontent.com/assets/8261945/9927239/cbcee2d4-5d14-11e5-9b28-4670ad24e19e.png">

I wonder if we can either change the color of this snapshat icon, or just have the linear ghost (will people pick it up ? ) or maybe put it in B&W with the name "snapchat" as you did For Facebook.

**Comment by cassiemc** [comment]
By pub I do mean publication. Like a special topic insert – "Deep dive on teenagers" with or as a separate document "Deep dive on informal educators." In my opinion the document is way too long as is, I worry about it being slightly repetitive, and some of the deeper points being lost in the mass. Those seem like sections ripe for pulling out as separate documents we can release later.... will get more attention that way. And then the main focus of this report becomes the Web Journey framework, the comparison with India, and the final recommendations.

Re. snapchat, will add it to my to-do list if there's time!

**Comment by cassiemc** [comment]
@LauraReynal @KevZawacki Is there a reason why 'dana boyd' isn't capitalized in the doc, or just an oversight?

**Comment by KevZawacki** [comment]
@cassiemc It's intentional — she styles her name that way.

**Comment by cassiemc** [comment]
huh, glad I asked. Thanks @KevZawacki!

**Comment by LauraReynal** [comment]
http://www.danah.org/name.html interesting :smile: @cassiemc

**Comment by cassiemc** [comment]
A few bits here and there left to do (section titles, TOC and a couple photo edits), but here are all the spreads (woohoo!). @LauraReynal could you review for any blockers wrt to content? @lovegushwa could you do a design review for me? Doesn't need to be **tooooo** thorough. 

@davidascher @KevZawacki you might be interested too.

https://drive.google.com/a/mozilla.com/file/d/0BzfN9sXlOXt1YlZpa3U5a2ZQM2s/view?usp=sharing

**Comment by lovegushwa** [comment]
@cassiemc yay, sure thing! :)

**Comment by lovegushwa** [comment]
@cassiemc  @LauraReynal R+ 

Looks great Cassie, I love the new quote styles and how you have clarified much of the content.
Most of the comments are about orphans (which is totally the last thing to fix in any doc). You can ignore what seems trivial, also there could be different schools of though about orphans on headlines so do as you think best! :)

https://drive.google.com/a/mozilla.com/file/d/0B2qu3ExKWIlvWDVORlpwLWI0SUU/view?usp=sharing

**Comment by LauraReynal** [comment]
@cassiemc  thanks this looks great :blush: 

R+ for content. 

The only thing I noted is that page 98 is the only quote of the report who is full page but different than the other full page quotes. Can it be the same as page 73 for example?

**Comment by cassiemc** [comment]
Good catch, @LauraReynal! @lovegushwa thanks for the R, should've mentioned I hadn't done typesetting yet! Thanks for the comments, immensely helpful :)

**Comment by cassiemc** [comment]
Ok, all done! Final pdfs.
 
* For online (lower res): https://drive.google.com/a/mozilla.com/file/d/0BzfN9sXlOXt1ZEN4eDlVTzcwa1E/view?usp=sharing
* For printing: 
https://drive.google.com/a/mozilla.com/file/d/0BzfN9sXlOXt1U3NyNWJYcVdpTXc/view?usp=sharing

Let me know if there is anything outstanding, otherwise should be good to publish @LauraReynal.

The complete folder with the indesign doc is here: https://drive.google.com/a/mozilla.com/folderview?id=0BzfN9sXlOXt1X200OHFQQkx6V1k&usp=sharing

Will reassign this bug to you Laura so you can close it once you publish the report.

**Comment by LauraReynal** [comment]
After 106 comments, I can't believe we are closing this ticket !!! 

****HUGE HUGE THANKS**** to everyone here who helped ! this was a big project and I am very grateful ! Thanks for the hard work :santa: :space_invader: :tulip: :flags: :dancer: :bouquet: :sun_with_face: :sunflower: :dancers: :candy: :baby: :bomb: :diamond_shape_with_a_dot_inside: :cat: :joy_cat: @cassiemc @lovegushwa @KevZawacki @thisandagain @xmatthewx @davidascher
